lib/mixlib/shellout.rb

```diff
@@ -234,6 +241,7 @@ def format_for_exception
       msg << "STDERR: #{stderr.strip}\n"
       msg << "---- End output of #{command} ----\n"
       msg << "Ran #{command} returned #{status.exitstatus}" if status
+      msg = "Command execution failed. STDOUT/STDERR suppressed for sensitive resource" if sensitive
```

Comment:
this can just be an early return; move to line 237 and just `return "Command execution failed. STDOUT/STDERR suppressed for sensitive resource" if sensitive`